src/feature-tracks.js

```diff
@@ -0,0 +1,49 @@
+//-------------------------------Feature Tracks logic starts------------------------------------//
+export const productsDOM = document.querySelector(".products-center");
+
+//getting the products
+export class Products {
+	async getProducts() {
+		try {
+			let result = await fetch("products.json");
+			let data = await result.json();
+
+			let products = data.items;
+			products = products.map((item) => {
+				const { title, price } = item.fields;
+				const { id } = item.sys;
+				const image = item.fields.image.fields.file.url;
+                const link = item.fields.link;
+				return { title, price, id, image, link };
+			});
+			return products;
+		} catch (error) {
+			console.log(error);
+		}
+	}
+}
```

Comment:
exporting function instead
```suggestion
export const getProducts = async() => {
		try {
			let result = await fetch("products.json");
			let data = await result.json();

			let products = data.items;
			products = products.map((item) => {
				const { title, price } = item.fields;
				const { id } = item.sys;
				const image = item.fields.image.fields.file.url;
                const link = item.fields.link;
				return { title, price, id, image, link };
			});
			return products;
		} catch (error) {
			console.log(error);
		}
	}
}
```